languages/elixir/exercises/concept/conditionals/.docs/instructions.md

```diff
@@ -0,0 +1,39 @@
+In this exercise you'll be mapping logging codes to a label.
+
+These are the different log codes:
+
+- 0 (trace)
+- 1 (debug)
+- 4 (info)
+- 5 (warning)
+- 6 (error)
+- 7 (fatal)
+
+You have two tasks:
+
+### 1. Return the logging code label
+
+Implement the `LogLevel.to_label/1` function to return the label of a log line from an integer code:
+
+```elixir
+LogLevel.to_label(1)
+# => :debug
+```
+
+### 2. Support unknown logging codes
+
+Unfortunately, some log lines occasionally have an unknown log code.  To gracefully handle these log lines, add an _unknown_ label to the `LogLevel.to_label/1` function.
+
+```elixir
+LogLevel.to_label(10)
+# => :unknown
+```
+
+### 3. Send an alert
+
+Implement the `LogLevel.send_alert?/1` function to determine if an alert should be send based on the log line's label.
```

Comment:
I might have missed it, but is it explained somewhere which log levels should have an alert sent?

Reply:
Will clarify

Reply:
Added in revision